common/display/displayqueue.cpp

```diff
@@ -111,6 +111,9 @@ bool DisplayQueue::Initialize(uint32_t width, uint32_t height, uint32_t pipe,
   drmModePropertyPtr broadcastrgb_props =
       drmModeGetProperty(gpu_fd_, broadcastrgb_id_);
 
+  if (!broadcastrgb_props)
+	  goto out;
```

Comment:
just return true here with a comment why this is still a valid case i.e. Its valid case that we might not be supporting this on certain panels. You are missing Jira: Tests: In commit message